src/platform/cmx/user/cos_component.h

```diff
@@ -0,0 +1,453 @@
+/**
```

Comment:
User is `components/implementation/...`.  We can't tear the directory structure apart.